java/compaction/compaction-status-store/src/main/java/sleeper/compaction/status/store/task/DynamoDBCompactionTaskStatusFormat.java

```diff
@@ -116,8 +116,8 @@ private static void addStatusUpdate(Map<String, AttributeValue> item, Compaction
                         .totalJobRuns(getIntAttribute(item, NUMBER_OF_JOBS, 0))
                         .totalRecordsRead(getLongAttribute(item, LINES_READ, 0))
                         .totalRecordsWritten(getLongAttribute(item, LINES_WRITTEN, 0))
-                        .recordsReadPerSecond(Double.parseDouble(getNumberAttribute(item, READ_RATE)))
-                        .recordsWrittenPerSecond(Double.parseDouble(getNumberAttribute(item, WRITE_RATE)))
+                        .recordsReadPerSecond(getDoubleAttribute(item, READ_RATE))
+                        .recordsWrittenPerSecond(getDoubleAttribute(item, WRITE_RATE))
```

Comment:
If we pass in the default value as a parameter, it would be more consistent with getLongAttribute and getIntAttribute. It also seems a bit clearer to show the default value explicitly, and more flexible to let you set it.